src/crimson/osd/ops_executer.h

```diff
@@ -152,6 +152,9 @@ class OpsExecuter {
   template <class Func>
   auto do_write_op(Func&& f, bool um) {
     ++num_write;
+    if (!osd_op_params) {
+      osd_op_params = osd_op_params_t{};
```

Comment:
```suggestion
      osd_op_params.emplace();
```

for saving the overhead of a move ctor of `osd_op_params_t`.